packages/server/src/types/json-schema.ts

```diff
@@ -41,19 +41,21 @@ export interface JsonSchemaNumber extends JsonSchemaPropertyBase {
 }
 
 export interface JsonSchemaObject extends JsonSchemaPropertyBase {
-  type: 'object'
-  additionalProperties?: JsonSchemaProperty
-  required?: string[]
-  propertyNames?: {
-    pattern: string
-  }
-  minProperties?: number
-  maxProperties?: number
-  properties?: {
-    [key: string]: JsonSchemaProperty
-  }
-  dependencies?: {
-    [key: string]: string[]
+  schema: {
```

Comment:
This isn't correct. This is the type for how to write a JSON object property using JSON Schema.